README.md

```diff
@@ -12,15 +12,16 @@ These reports are incubated by the [Solid Notifications Panel](https://github.co
 The Solid Notification Protocol makes it possible to define any number of subscription types.
 The list of subscription types below does not intend to be comprehensive. This list is intended
 to be useful to application developers looking for existing subscription types as well as to
-authors of new subscription types. New subscription types may be added here, but there is no
-requirement to do so.
+authors of new subscription types.
 
 * [WebSocketSubscription2021](https://solid.github.io/notifications/websocket-subscription-2021) Editor's Draft.
 * [StreamingHTTPSubscription2021](https://solid.github.io/notifications/streaming-http-subscription-2021) Editor's Draft.
 * [EventSourceSubscription2021](https://solid.github.io/notifications/eventsource-subscription-2021) Editor's Draft.
 * [LinkedDataNotificationsSubscription2021](https://solid.github.io/notifications/linkeddatanotifications-subscription-2021) Editor's Draft.
 * [WebHookSubscription2021](https://github.com/solid/notifications/blob/main/webhook-subscription.md) Editor's Draft.
 
+New subscription types may be added here, but there is no requirement to do so. Adding a subscription type to the list above aids in its discoverability by the solid community. We request authors of new subscription types to open a PR with this repository, so that the subscription type they have authored can be included in the list above.
```

Comment:
There is a bit of redundancy (and also with the earlier paragraph). What do you think of this:

```suggestion
To add a new subscription type to the list, please create a PR including the name and the URL of the document describing the subscription type. The Solid CG reserves the right to include or update the list.
```

Reply:
I like your language better!

Reply:
Now that I think about it, would we like to have a leading line saying something like,"We welcome and encourage community contributions of subscription types."

Reply:
I think that's implied by "to add a new subscription.."